### PR TITLE
feat(sql): fix display of interval scans in EXPLAIN's output

### PR DESCRIPTION
**Pull request by bziobrowski**
Changes : 
- fix display for interval scans with dynamic intervals 
- show iso-formatted timestamps instead of raw long values

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 21 / 23 (91.30%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/model/RuntimeIntervalModel.java|19|21|90.48%|
|:large_blue_circle:|io/questdb/griffin/BasePlanSink.java|2|2|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 21 / 23 (91.30%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/model/RuntimeIntervalModel.java|19|21|90.48%|
|:large_blue_circle:|io/questdb/griffin/BasePlanSink.java|2|2|100.00%|

File: core/src/test/java/io/questdb/test/griffin/ExplainPlanTest.java [hunk 1]
```diff
@@ -4668,7 +4676,7 @@ public void testSelectDynamicTsInterval5() throws Exception {
                 "DataFrame\n" +
                         "    Row forward scan\n" +
                         "    Interval forward scan on: tab\n" +
-                        "      intervals: [static=[0,9223372036854775807,281481419161601,4294967296,1640995200000001,9223372036854775807,2147483649,4294967296] dynamic=[now(),null]]\n");
+                        "      intervals: [(\"2022-01-01T00:00:00.000001Z\",\"MAX\")]\n");
```
[the remark this answers is not in the document]
Comment (bziobrowski) [reply core/src/test/java/io/questdb/test/griffin/ExplainPlanTest.java:4679]: No, interval scan now shows concrete intervals to scan so now()` gets replaced with current timestamp . 
I think it's better than showing both static and dynamic intervals and watching out for the the different internal representations of intervals

**Review by ideoma — APPROVED** [review]

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 21 / 23 (91.30%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/model/RuntimeIntervalModel.java|19|21|90.48%|
|:large_blue_circle:|io/questdb/griffin/BasePlanSink.java|2|2|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 21 / 23 (91.30%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/model/RuntimeIntervalModel.java|19|21|90.48%|
|:large_blue_circle:|io/questdb/griffin/BasePlanSink.java|2|2|100.00%|

**Comment by bluestreak01** [comment]
@bziobrowski another one with incorrect prefix

**Pull request merged**